code/game/gamemodes/cult/blood_magic.dm

```diff
@@ -765,6 +810,14 @@
 		//Draining blood on the floor
 		if(istype(target, /obj/effect/decal/cleanable/blood) || istype(target, /obj/effect/decal/cleanable/trail_holder))
 			blood_draw(target, user)
+		if(istype(target, /obj/item/blood_orb))
+			var/obj/item/blood_orb/candidate = target
+			if(candidate.blood)
+				uses += candidate.blood
+				to_chat(user, "<span class='warning'>You obtain [candidate.blood] from the orb of blood!</span>")
```

Comment:
Should this be 
```suggestion
				to_chat(user, "<span class='warning'>You obtain [candidate.blood] blood from the orb of blood!</span>")
```
? 
So that it will read "You obtain 5 blood from the orb..." instead of "You obtain 5 from the orb..."